apps/tools/librelingo_tools/lili.py

```diff
@@ -135,8 +135,8 @@ def export_main_html_page(course, count, reldir, html_dir):
         course=course,
         repository_url=get_repository_url(course),
     )
-    with open(os.path.join(html_dir, "modules.html"), "w") as fh:
-        fh.write(html)
+    with open(os.path.join(html_dir, "modules.html"), "w") as f_h:
```

Comment:
this is the same as the case that I mentioned above, I I think this is just a file pointer, it could be named with either a custom name that is appropriate for this use case, or a generic name like you did above (`file` or `file_pointer`)